src/components/propositionalLogic/examplePage.tsx

```diff
@@ -2,75 +2,72 @@ import React from "react";
 
 import styles from "./propositionalLogic.module.scss";
 
+const exampleProofList: Array<Array<[string, string, boolean]>> = [
+	[
+		["Wir zeigen: a -> (b -> c) -> (a und b) -> c.", "Dies soll bewiesen werden.", false],
+		["Es seien a, b und c Aussagen.", "Allgemeine Vorraussetzung", false],
+		["Angenommen $a ->(b -> c)$.", "Vorraussetzung: Die Prämisse a -> (b -> c) ist wahr.", false],
+		["Angenommen ferner es gilt $(a und b)$.", "Vorraussetzung: (a und b) ist wahr.", false],
+		["Dann folgt a.", "a folgt, da (a und b) nur dann wahr ist, wenn a wahr ist.", false],
+		["Ausserdem folgt b.", "b folgt, da (a und b) nur dann wahr ist, wenn b wahr ist.", false],
+		["Damit gilt $(b -> c)$.", "Die Konlusion (b -> c) gilt, da die Prämissen a,sowie (a und b) beide gelten.", false],
+		["Ferner folgt c.", "Die Konlusion c gilt, da die Prämissen a, sowie (a und b) beide gelten.", false],
+		["Also gilt $a -> (b -> c) -> (a und b) -> c$.", "qed", false]],
+	[
+		["test1", "test2", false],
+		["test3", "test4", false]],
+];
+
 interface State {
-	proofPart: Array<[string, string, boolean]>;
+	exampleProofList: Array<Array<[string, string, boolean]>>;
 }
 
 class ExamplesPropositionalLogic extends React.Component<{}, State> {
 
 	// Via constructor since an immediate assignment led to type issues
 	constructor(props: {}) {
 		super(props);
-		const proofPart: Array<[string, string, boolean]> = [
-			["Es seien a, b und c Aussagen.", "Allgemeine Vorraussetzung", false],
-			["Angenommen $a ->(b -> c)$.", "Vorraussetzung: Die Prämisse a -> (b -> c) ist wahr.", false],
-			["Angenommen ferner es gilt $(a und b)$.", "Vorraussetzung: (a und b) ist wahr.", false],
-			["Dann folgt a.", "a folgt, da (a und b) nur dann wahr ist, wenn a wahr ist.", false],
-			["Ausserdem folgt b.", "b folgt, da (a und b) nur dann wahr ist, wenn b wahr ist.", false],
-			["Damit gilt $(b -> c)$.", "Die Konlusion (b -> c) gilt, da die Prämissen a,sowie (a und b) beide gelten.", false],
-			["Ferner folgt c.", "Die Konlusion c gilt, da die Prämissen a, sowie (a und b) beide gelten.", false],
-		];
 
-		this.state = { proofPart };
+		this.state = { exampleProofList };
 	}
 
 	public render() {
 		return <div className={styles.site}>
-			<table>
+			<button>Nächstes Beispiel</button>
+			<table className={styles.example}>
 				<tbody>
 					<tr>
 						<th>Eingabe</th>
-						<th></th>
-					</tr>
-					<tr>
-						<th>Wir zeigen: a -> (b -> c) -> (a und b) -> c.</th>
-						<th>Dies soll bewiesen werden. </th>
+						<th>Klicke auf "Hilfe", um eine Erklärung anzeigen zu lassen.</th>
 					</tr>
 					<tr>
 						<td></td>
 						<td></td>
 					</tr>
-					{this.renderProof()}
-					<tr>
-						<td></td>
-						<td></td>
-					</tr>
-					<tr>
-						<td>Also gilt $a -> (b -> c) -> (a und b) -> c$.</td>
-						<td>qed </td>
-					</tr>
+					{this.renderProof(0)}
```

Comment:
You can use `this.state.exampleToShow` as a parameter then or just use it in the function and get rid of the parameter.